src/main/java/com/thangle/error/AccessDeniedException.java

```diff
@@ -0,0 +1,10 @@
+package com.thangle.error;
+
+import org.springframework.http.HttpStatus;
+
+public class AccessDeniedException extends DomainException {
+
+    public AccessDeniedException(final Object... args) {
```

Comment:
Cố gắng để ý code từng cái dùng để làm gì, ví dụ ở đây args hoàn toàn không có ý nghĩa


```suggestion
    public AccessDeniedException() {
```